src/snapred/ui/workflow/DiffCalWorkflow.py

```diff
@@ -372,6 +374,7 @@ def _saveCalibration(self, workflowPresenter):
         # pull fields from view for calibration save
         calibrationIndexEntry = CalibrationIndexEntry(
             runNumber=view.fieldRunNumber.get(),
+            useLiteMode=view.useLiteMode,
```

Comment:
I think this could use `self.useLiteMode` instead and avoid the use of signals.  But I did not want to risk it.